components/airlock/src/user.rs

```diff
@@ -45,11 +45,15 @@ pub fn home_dir_for_username(username: &str) -> Result<PathBuf> {
 }
 
 pub fn my_username() -> Result<String> {
-    users::get_effective_username().ok_or(Error::EffectiveUsernameNotFound)
+    users::get_effective_username()
```

Comment:
Since the code to convert from OsString → String [already exists in the core crate](https://github.com/habitat-sh/core/blob/master/components/core/src/os/users/linux.rs#L48), why not use it?

Reply:
I suspect there may have been a desire to not depend on any Habitat-specific code for `airlock`, though @fnichol would know that for sure.